### ISSUE TITLE
new button links on the front page

### ISSUE THREAD

**Issue by AmandaBekhoj** [mamont77/skillspeople#132]
So, as you can probably gather, we are changing our business a bit. which means all these changes to the site. 

We need two buttons on the front page (located where the social media icons where before, i have removed them)

One of the buttons should say 'Søg Job' and link to the new search interface that you are building right now, and the other button should say 'Opret announce' which should like to a form that we have yet to create. 

![skaermbillede 2015-02-05 kl 13 27 03](https://cloud.githubusercontent.com/assets/8081414/6060316/fbf09c0c-ad3b-11e4-8886-131eabd91b97.png)

also, this search function should not be located here, it is weird. could you remove it? :)
![skaermbillede 2015-02-05 kl 13 26 43](https://cloud.githubusercontent.com/assets/8081414/6060343/3c81a1a8-ad3c-11e4-9d72-01c95eb4961b.png)

**Comment by mamont77** [comment]
>>and the other button should say 'Opret announce' which should like to a form that we have yet to create.

Please create a new issue for this task.

**Comment by mamont77** [comment]
This task depends on the #127 and #136.

**Comment by AmandaBekhoj** [comment]
yes, #127 should be finished first!

**Comment by mamont77** [comment]
Good Morning Amanda.
Please see pictures. This is on my local PC. If that's what you want - I push the site.

One btn.
![skillspeople - referral recruitment kobenhavn - google chrome 02-13-2015 08 51 52](https://cloud.githubusercontent.com/assets/970967/6183521/d904235c-b35e-11e4-811c-dece97f873ba.png)

Two btns (Be included later).
![skillspeople - referral recruitment kobenhavn - google chrome 02-13-2015 08 52 20](https://cloud.githubusercontent.com/assets/970967/6183532/f32d1194-b35e-11e4-9254-2c799d0d8ef4.png)

Mobile devices.
![skillspeople - referral recruitment kobenhavn - google chrome 02-13-2015 08 52 40](https://cloud.githubusercontent.com/assets/970967/6183542/05578f16-b35f-11e4-9241-ffe56a00dc84.png)
 
P.S.: Some pictures are lost, do not pay attention. It's my local sandbox.

**Comment by AmandaBekhoj** [comment]
Good morning!
It looks exactly like a imagined. but do you think you could make the buttons the same orange/yellow as we have in the slider? and then with white font :)

also, do you think you could move the buttons just a bit more down? they are placed a little to close to the logo-block :)

**Comment by mamont77** [comment]
Done.
![skillspeople - referral recruitment kobenhavn - google chrome 02-13-2015 11 49 29](https://cloud.githubusercontent.com/assets/970967/6185389/7851a026-b376-11e4-82cb-fced92e862ff.png)
Can i push to live site?

**Comment by mamont77** [comment]
Pushed together with #142

**Comment by AmandaBekhoj** [comment]
yes, you can push to live site :+1:

**Comment by mamont77** [comment]
Done.

**Comment by AmandaBekhoj** [comment]
sorry for reopening this issue - i wanted to change the name of the button, but i cannot find out where to do?

also, my colleague experience some difficulties with the site, he sent me this: 
![unknown](https://cloud.githubusercontent.com/assets/8081414/6210959/2ccc1602-b5d5-11e4-9903-c36ec0c857ae.png)

but everything works fine on my Mac, so i do not know what this is?

**Comment by mamont77** [comment]
>> also, my colleague

I pushed the new files to the site. It lasted no more than 2 seconds. My apologies.

>> i wanted to change the name of the button

It's hardcoded in the code, send me the name of the button.

**Comment by AmandaBekhoj** [comment]
No problem, i will ask him to try again :)

name of button 'Find Job' - i would not want to mess with hardcore code!

**Comment by mamont77** [comment]
>>i would not want to mess with hardcore code!

hardcode != hardcore :-)

ok, 'll see how it migrate to the content (as a normal block).

**Comment by mamont77** [comment]
Fixed.
http://www.skillspeople.dk/admin/structure/block/manage/block/36/configure?destination=front

**Comment by AmandaBekhoj** [comment]
hahah sorry, i read too quick :b 

thanks!
